sdk/resourcemanager/Azure.ResourceManager/tests/Scenario/ResourceGroupOperationsTests.cs

```diff
@@ -101,6 +105,14 @@ public async Task Update()
             Assert.AreEqual(rg1.Data.ManagedBy, rg2.Data.ManagedBy);
             Assert.AreEqual(rg1.Data.Tags, rg2.Data.Tags);
 
+            var parameters3 = new ResourceGroupPatch
+            {
+                Name = rgName
+            };
+            parameters3.Tags.Clear();
```

Comment:
Do we have to use this clear to make the patch work?

Reply:
Yes. The first update should keep tags unchanged:
https://github.com/Azure/azure-sdk-for-net/blob/bbe4fdec6e35db7b6c3f27d827e6701293e6779e/sdk/resourcemanager/Azure.ResourceManager/tests/Scenario/ResourceGroupOperationsTests.cs#L95-L99

The second update should remove all tags:
https://github.com/Azure/azure-sdk-for-net/blob/bbe4fdec6e35db7b6c3f27d827e6701293e6779e/sdk/resourcemanager/Azure.ResourceManager/tests/Scenario/ResourceGroupOperationsTests.cs#L108-L113

Previously, I believe both update will wipe out all tags which makes the first update causing surprises.

Reply:
I tried the following and it  will not remove the tags as the ChangeTrackingDictionary is still considered undefined:
```
var parameters3 = new ResourceGroupPatch
{
    Name = rgName,
    Tags = {}
};
ResourceGroupResource rg3 = await rg2.UpdateAsync(parameters3);
```